src/scripts/timezone.test.js

```diff
@@ -127,67 +131,101 @@ describe("Handy Tau-bot timezone conversions", () => {
       await handler(message);
 
       expect(slack.postEphemeralMessage).toHaveBeenCalledWith(
-        responseFor("user 1", "1:00")
+        responseFor("user dn", "1:00")
       );
 
       expect(slack.postEphemeralMessage).toHaveBeenCalledWith(
-        responseFor("user 4", "11:00")
+        responseFor("user ak", "11:00")
       );
 
-      expect(slack.postEphemeralMessage.mock.calls.length).toEqual(2);
+      expect(slack.postEphemeralMessage.mock.calls.length).toEqual(3);
     });
 
     it("if a timezone is not specified, it messages everyone except the original author", async () => {
       message.event.text = "03:00";
       await handler(message);
 
       expect(slack.postEphemeralMessage).toHaveBeenCalledWith(
-        responseFor("user 1", "2:00")
+        responseFor("user ny", "4:00")
+      );
+
+      expect(slack.postEphemeralMessage).toHaveBeenCalledWith(
+        responseFor("user ch", "3:00")
       );
 
       expect(slack.postEphemeralMessage).toHaveBeenCalledWith(
-        responseFor("user 2", "4:00")
+        responseFor("user dn", "2:00")
       );
 
       expect(slack.postEphemeralMessage).toHaveBeenCalledWith(
-        responseFor("user 3", "3:00")
+        responseFor("user la", "1:00")
       );
 
       expect(slack.postEphemeralMessage).toHaveBeenCalledWith(
-        responseFor("user 4", "12:00")
+        responseFor("user ak", "12:00")
       );
 
       // Make sure only the above messages were posted.
-      expect(slack.postEphemeralMessage.mock.calls.length).toEqual(4);
+      expect(slack.postEphemeralMessage.mock.calls.length).toEqual(5);
     });
 
     it("includes AM/PM in the message if it was included in the original", async () => {
       message.event.text = "03:00 PM";
       await handler(message);
 
       expect(slack.postEphemeralMessage).toHaveBeenCalledWith(
-        responseFor("user 1", "2:00 pm")
+        responseFor("user dn", "2:00 pm")
       );
 
       // Not testing filtering here, so trust the previous tests.
     });
 
+    it("correctly includes AM/PM when crossing noon", async () => {
```

Comment:
This is a good addition. These tests should make it safer to eventually migrate away from moment.js.